### ISSUE TITLE
[commonjs] require.resolve is rewritten

### ISSUE THREAD

**Issue by tkalliom** [rollup/plugins#406]
- Rollup Plugin Name: commonjs
- Rollup Plugin Version: 12.0.0
- Rollup Version: 2.10.7
- Operating System (or Browser): Linux
- Node Version: 10.19.0

### How Do We Reproduce?

See https://repl.it/repls/RobustConcreteHypertalk for a complete example.

Module:
```
module.exports.getLocation = function() {
  return require.resolve("./rollup.config.json");
}
```

Input:
```
import { getLocation } from "locator";

console.log("Location of config is " + getLocation());
```

### Expected Behavior
Rollup leaves `require.resolve` as-is, because no actual requiring is performed.

### Actual Behavior
Rollup replaces `require` with `commonjsRequire`, causing the error `commonjsRequire.resolve is not a function`.

**Comment by thecotne** [comment]
as a workaround you can replace `commonjsRequire.resolve` with `require.resolve` from output plugins

```js
export default {
  // ...
  output: {
    // ...
    plugins: [
      replace({ 'commonjsRequire.resolve': 'require.resolve' })
    ]
  }
}
```

**Comment by tkalliom** [comment]
This was fixed by #446
